Jenkinsfile

```diff
@@ -866,7 +866,7 @@ pipeline {
             }
         }
         stage("MIOpenTensile"){
-            when { expression { params.BUILD_CURRENT_STAGE } }
+            when { expression { !params.BUILD_CURRENT_STAGE } }
```

Comment:
Which means: "If current stage build is disabled, then build current stage". :joy: :joy: :joy:

But no problem, we'll fix that later.

Reply:
Sorry I don't get it. What is the issue here?

Reply:
No issue, except the code is obfuscated ;) Don't mind.